lib/fluent/supervisor.rb

```diff
@@ -109,6 +127,11 @@ def run_rpc_server
         end
         nil
       }
+      @rpc_server.mount_proc('/api/processes.zeroDowntimeRestart') { |req, res|
+        $log.debug "fluentd RPC got /api/processes.zeroDowntimeRestart request"
+        Process.kill :USR2, Process.pid
+        nil
+      } unless Fluent.windows?
```

Comment:
```suggestion
      unless Fluent.windows?
        @rpc_server.mount_proc('/api/processes.zeroDowntimeRestart') { |req, res|
          $log.debug "fluentd RPC got /api/processes.zeroDowntimeRestart request"
          Process.kill :USR2, Process.pid
          nil
        }
      end
```

Reply:
Thanks!

`/api/config.getDump` has the same format, so I fixed this together.
https://github.com/fluent/fluentd/compare/c714d9ca10542cccafc01b010a784c897f9c4954..a6a14de66001e39415e4537dc0b85702b5076988